jkoenig's work on signals

This dates when the global signal disposition call was added.

---
 Versions.def  |    1 +
 hurd/Versions |    8 ++++++++
 2 files changed, 9 insertions(+)

--- a/Versions.def
+++ b/Versions.def
@@ -41,6 +41,7 @@ libc {
   GCC_3.0
 %endif
   GLIBC_PRIVATE
+  GLIBC_2.13_DEBIAN_19
 }
 libcrypt {
   GLIBC_2.0
--- a/hurd/Versions
+++ b/hurd/Versions
@@ -152,6 +152,14 @@ libc {
     # functions used in macros & inline functions
     __errno_location;
   }
+  GLIBC_2.13_DEBIAN_19 {
+    # functions used by libpthread and <hurd/signal.h>
+    _hurd_sigstate_set_global_rcv;
+    _hurd_sigstate_lock;
+    _hurd_sigstate_pending;
+    _hurd_sigstate_unlock;
+    _hurd_sigstate_delete;
+  }
 
 %if !SHLIB_COMPAT (libc, GLIBC_2_0, GLIBC_2_2)
   HURD_CTHREADS_0.3 {
